### ISSUE TITLE
::-webkit-scrollbar  not working since a few days

### ISSUE THREAD

**Issue by montsea999** [mdn/browser-compat-data#22195]
### What type of issue is this?

Other

### What information was incorrect, unhelpful, or incomplete?

In my angular app, I have a custom scroll, but since a few days, the custom style
 del ::-webkit-scrollbar stopped applying and I only see a default scrollbar. My source code has not been changed

### What did you expect to see?

The scroll bar as before. If this is not possible, at least be able to hide the arrow buttons

### Did you test this? If so, how?

 I have done several tests: 
If I go to an old commit I also see the default bar. 
If I go to any of the environments I also see the default bar. 

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `css.selectors.-webkit-scrollbar`
* Report started: 2024-02-10T22:43:28.022Z

</details>

**Comment by squivix** [comment]
I have the same problem on my Vue SPA web app.

**Comment by lukewarlow** [comment]
If you're using Chromium (Chrome/Edge etc) this is expected behaviour, as of Chrome 121 the standard properties are supported and as always the standard CSS overrides the non-standard.

I don't believe this warrants any changes to the BCD entry but perhaps a note on the content page itself would be useful?

**Comment by mcroba** [comment]
Thank you for your feedback @lukewarlow. 

My main issue is that with Chrome and Edge supporting now `scrollbar-*`, we start to see arrows in our custom scrollbar (which was not the case with webkit-scrollbar), so it's kind of a visual regression.

**Comment by lukewarlow** [comment]
There's a better way to conditionalise them than detecting Firefox.

@supports not (::-webkit-scrollbar) { } should do the trick (not checked exact syntax) in a more future proof way.

**Comment by mcroba** [comment]
Yes I finally went for that (similarly as mentioned [here](https://github.com/mdn/browser-compat-data/issues/22195#issuecomment-1938143402))

```
@supports not selector(::-webkit-scrollbar) {}
```

Hopefully, at some point, we will have a way to remove the arrows with `scrollbar-*`

**Comment by montsea999** [comment]
The way with the support did not work for me and I found this other way that solved it for me. 
Apply the way that works for you while the problem is fixed ;)

**Comment by lukewarlow** [comment]
Are you able to close this issue as it's not a probably with the compat data, I've raised a separate content issue to make a more prominent note on the page itself.
